tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php

```diff
@@ -58,14 +58,30 @@ public function testCacheTimeout()
 	{
 		/** @var Cache_Lite $cacheLiteInstance */
 		$cacheLiteInstance = TestReflection::getValue('JCacheStorageCachelite', 'CacheLiteInstance');
-		$cacheLiteInstance->_lifeTime = 2;
+		$cacheLiteInstance->_lifeTime = 0.1;
 
 		$data = 'testData';
 
 		$this->assertTrue($this->handler->store($this->id, $this->group, $data), 'Initial Store Failed');
 
-		sleep(3);
+		// Timer and max time (in seconds)
+		$timer    = 0;
+		$maxTime  = 3;
 
-		$this->assertFalse($this->handler->get($this->id, $this->group), 'No data should be returned from the cache store when expired.');
+		// Testing interval (in seconds)
+		$interval = 0.1;
+
+		do
+		{
+			$timer += $interval;
+
+			usleep($interval * 1000000);
+
+			$cache = $this->handler->get($this->id, $this->group);
+
```

Comment:
How many loops is executed? I think that first loop should return valid data, second may return false.
Then we will be sure that cache works at all.